src/client.py

```diff
@@ -202,6 +204,11 @@ def reprocess(self, start_date=None, end_date=None):
 
     ###########################################################
 
+    def create_all_views(self):
+        self.flagged.create_views_all_flags()
+
```

Comment:
If you could add this to the client documentation in the README that'd be much appreciated.